DotNet/emitter.js

```diff
@@ -338,6 +348,19 @@
 			addCommand(dot_net`webDriver = eyes.Open(driver, ${appNm}, ${test.config.baselineName}` + rectangle + ');')
 		},
 		check(checkSettings = {}) {
+			if(checkSettings !== undefined && checkSettings.visualGridOptions)
+			{
+				if (!confVisualGridOptionCreated) {
+					addCommand(`var conf = eyes.GetConfiguration();`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
					addCommand("var conf = eyes.GetConfiguration();")
```